lib/web/websocket/frame.js

```diff
@@ -2,26 +2,43 @@
 
 const { maxUnsigned16Bit } = require('./constants')
 
+const BUFFER_SIZE = 16386
+
 /** @type {import('crypto')} */
 let crypto
+let buffer = null
+let bufIdx = 0
+
 try {
   crypto = require('node:crypto')
 /* c8 ignore next 3 */
 } catch {
 
 }
 
+function generateMask () {
+  if (buffer === null) {
+    buffer = crypto.randomFillSync(Buffer.allocUnsafe(BUFFER_SIZE), 0, BUFFER_SIZE)
+  }
+  if (bufIdx + 4 > BUFFER_SIZE) {
```

Comment:
why though?

why not (if bufIdx === BUFFER_SIZE) { ?

Reply:
Thanks!